### ISSUE TITLE
Not all images have alt text

### ISSUE THREAD

**Issue by jeremyfelt** [washingtonstateuniversity/murrow.wsu.edu#124]
![screen shot 2017-09-12 at 11 20 08 am](https://user-images.githubusercontent.com/286171/30341863-8410118c-97ac-11e7-8ed4-81202a92a241.png)

```
<a href="https://s3.wp.wsu.edu/uploads/sites/880/2017/08/MurrowDSC_6242.jpg"><img class=" wp-image-1865 alignright" src="https://s3.wp.wsu.edu/uploads/sites/880/2017/08/MurrowDSC_6242-396x228.jpg" alt="" width="490" height="282" srcset="https://s3.wp.wsu.edu/uploads/sites/880/2017/08/MurrowDSC_6242-396x228.jpg 396w, https://s3.wp.wsu.edu/uploads/sites/880/2017/08/MurrowDSC_6242-768x442.jpg 768w, https://s3.wp.wsu.edu/uploads/sites/880/2017/08/MurrowDSC_6242-792x455.jpg 792w, https://s3.wp.wsu.edu/uploads/sites/880/2017/08/MurrowDSC_6242-990x569.jpg 990w, https://s3.wp.wsu.edu/uploads/sites/880/2017/08/MurrowDSC_6242-1188x683.jpg 1188w, https://s3.wp.wsu.edu/uploads/sites/880/2017/08/MurrowDSC_6242.jpg 1200w" sizes="(max-width: 490px) 100vw, 490px"></a>
```

**Comment by jeremyfelt** [comment]
This is the case for several images pulled in through content syndicate as well that have no alt text.

**Comment by jeremyfelt** [comment]
This is also the case on other pages - https://stage.murrow.wsu.edu/about/

In general, not all images have alt text.
